conjure-runtime/src/blocking/client.rs

```diff
@@ -20,9 +20,14 @@ use http::Method;
 ///
 /// It implements the Conjure `Client` trait, but also offers a "raw" request interface for use with services that don't
 /// provide Conjure service definitions.
-#[derive(Clone)]
```

Comment:
Why doesn't the derive work here?

Reply:
The derived implementation requires `T: Clone` which we don't require. It happened to work just fine before because `T` always had to be `Clone` anyway, but that's no longer the case.